lib/hyrax/specs/shared_specs/hydra_works.rb

```diff
@@ -291,3 +290,13 @@
     end
   end
 end
+
+RSpec.shared_examples 'a Hyrax::ChangeSet' do
```

Comment:
should this get organized into a different file? `lib/hyrax/specs/shared_specs/change_set.rb`?

Reply:
Good idea.  DONE